bin/cli.py

```diff
@@ -1083,9 +1104,11 @@ def __init__(self,):
         self.license = LicenseProxy()
         self.httpclient = HttpClientProxy()
         self.user = UserProxy()
+        self.role = RoleProxy()
         self.autocomplete = AutoComplete()
         self.configure_cli = configure_cli
 
 
+
```

Comment:
Too many blank lines, there shouldn't be more than 2.

Reply:
Got it!
Thank you :")